tests/test_emission_calculation.py

```diff
@@ -0,0 +1,119 @@
+import datetime
+
+import pytest
+from qgis.core import QgsMapLayer, QgsVectorLayer
+from qgis.testing import start_app
+from qgis.testing.mocked import get_iface
+
+from open_alaqs.core.alaqsdblite import ProjectDatabase
+from open_alaqs.core.modules.ModuleManager import OutputAnalysisModuleRegistry
+from open_alaqs.openalaqsdialog import OpenAlaqsResultsAnalysis
+from tests.utils import get_copy_path, get_data_path
+
+start_app()
+
+
+@pytest.fixture(scope="module")
+def plugin_instance():
+    print("\nINFO: Get plugin instance")
```

Comment:
Is there a strong reasoon to prefer `print` over `logging`? I think `logging` shows a better purpose on why it exists.

Reply:
TIL, thanks